frontend/package.json

```diff
@@ -73,12 +75,14 @@
         "@types/node": "^16.11.7",
         "@types/react": "^17.0.35",
         "@types/react-dom": "^17.0.11",
+        "@types/string-similarity": "^4.0.0",
         "@types/react-router-dom": "^5.3.2",
         "@types/testing-library__jest-dom": "^5.14.1",
         "@types/validator": "^13.7.0",
         "@types/webpack-dev-server": "^4.5.0",
         "@typescript-eslint/eslint-plugin": "^5.4.0",
         "@typescript-eslint/parser": "^5.4.0",
+        "ajv": "^8.6.3",
```

Comment:
Are you sure ajv, deep-diff, diff, and string-similarity are dev dependencies?

Reply:
fixed